index.ts

```diff
@@ -204,14 +206,14 @@ export default function generate(options: Options): Promise<void> {
 	filenames.forEach(name => { verboseMessage('  ' + name); });
 	const excludesMap: { [filename: string]: boolean; } = {};
 
-	options.exclude = options.exclude || [ 'node_modules/**/*.d.ts' ];
+	options.exclude = getFilenames(baseDir, options.exclude || [ 'node_modules/**/*.d.ts' ]);
 
-	options.exclude && options.exclude.forEach(function (filename) {
-		glob.sync(filename).forEach(function(globFileName) {
-			excludesMap[filenameToMid(pathUtil.resolve(baseDir, globFileName))] = true;
-		});
+	options.exclude.forEach(function (filename) {
```

Comment:
I believe the original options.exclude && was in case node_modules contained no *.d.ts files? Seems unlikely to have that issue though?